core/input/input_event.h

```diff
@@ -526,8 +549,8 @@ class InputEventMIDI : public InputEvent {
 	void set_channel(const int p_channel);
 	int get_channel() const;
 
-	void set_message(const MIDIMessage p_message);
-	MIDIMessage get_message() const;
+	void set_message(const Message p_message);
```

Comment:
Does this actually work? The enum values are the same but it's a different enum so it shouldn't match in GDScript and elsewhere, right?

Reply:
Not sure what you mean by "work". When I excluded static methods, the engine seems to compile just fine when replacing the **MIDIMessage** enum with **Message** all throughout.

Reply:
Just verified in my current build (without the static methods because I must reiterate, do not compile). Yes, you can get and set the event's `message` just fine.

Reply:
That's strange, because assigning other, matching valued, enums in GDScript like that doesn't work correctly, like doing:
```gdscript
event.messsage = TEXTURE_REPEAT_PARENT_NODE
```
Throws an error as the result type is not the correct enum

Reply:
That would be a breaking change btw, even if the types match I'd say

Reply:
I'm not sure changing the enum here preserves compatibility, we'll see what the CI says

Reply:
Fixed up and built your code and can confirm that this does indeed not work:
```gdscript
event.message = MIDI_MESSAGE_NONE
```
It has to be:
```gdscript
event.message = InputEventMIDI.MESSAGE_NONE
```

This breaks compatibility unless you change the bound method to take the original type